src/server/supplements/recorder/javascript.ts

```diff
@@ -84,8 +93,12 @@ export class JavaScriptLanguageGenerator implements LanguageGenerator {
 
     if (emitPromiseAll)
       formatter.add(`]);`);
-    else if (signals.assertNavigation)
-      formatter.add(`  // assert.equal(${pageAlias}.url(), ${quote(signals.assertNavigation.url)});`);
+    else if (signals.assertNavigation) {
+      if (this._isTest)
+        formatter.add(`  expect(${pageAlias}.url()).toBe(${quote(signals.assertNavigation.url)});`);
+      else
+        formatter.add(`  // assert.equal(${pageAlias}.url(), ${quote(signals.assertNavigation.url)});`);
```

Comment:
Maybe remove the asserts? We have tests with expect for people now.

Reply:
Yes, I did it and then test started failing and i don't want to fix it for merge. Will follow up.